cmd/kube-scheduler/app/options/options.go

```diff
@@ -169,11 +169,9 @@ func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 func (o *Options) ApplyTo(c *schedulerappconfig.Config) error {
 	if len(o.ConfigFile) == 0 {
 		c.ComponentConfig = o.ComponentConfig
+		o.Deprecated.ApplyProfileTo(&c.ComponentConfig)
 
 		// only apply deprecated flags if no config file is loaded (this is the old behaviour).
```

Comment:
Still call the function after the comment, but remove the word "only" from it.

Reply:
done